### PR TITLE
Improved logging with react-inspector

### PR DESCRIPTION
**Pull request by ritz078**
Key points : 
* Removed action name from logged object and showing it separately on right.
  - Generally showing it separately from Object is what I expected as that's not part of my Component.
  - In the current log method, if name remained then the logged object would have always looked like `Object{name: 'foo', args: Array[x]}` which would have looked same for every log thus making it hard to see the difference between different logs until expanded.

* showing `__proto__`.
 - It won't cause any perf issue as it is created inside the Inspector Component and isn't a part of `action` object.

### CODE REVIEW

**Comment by arunoda** [comment]
This is pretty cool and I'm getting this now.

**Comment by arunoda** [comment]
All these changes comes with `@kadira/storybook@1.37.0`.
Thanks again :)

**Comment by xyc** [comment]
Hi, react-inspector author here :) Thanks for using this lib!

Just a friendly reminder that an upcoming change is that the `name` prop will be rendered in the root node as per issue:
https://github.com/xyc/react-inspector/issues/13
(Currently the `name` prop **has no effects**)

So the result for `<Inspector name="test" data={{k: 'v'}}/>` will look like
```
 ▶ test: Object {k: 'v'}
```

So you might want to remove the name prop if you don't want it to be shown on the left side:
```js
+          <Inspector
+            showNonenumerable
+            data={action.data.args || action.data}
+          />
```

**Comment by ritz078** [comment]
@xyc Thats nice. In a way we would love that feature.

**Comment by arunoda** [comment]
I agree with @ritz078. 
@xyc The new one is way better :)

File: src/modules/ui/components/action_logger/index.js [hunk 1]
```diff
@@ -34,12 +34,60 @@ const btnStyle = {
...
+          { action.count > 1 && <span style={ countStyle }>{ action.count }</span> }
+        </div>
+        <div style={inspectorStyle}>
+          <Inspector
+            showNonenumerable
+            name={action.data.name}
+            data={action.data.args || action.data}
+          />
+        </div>
+        <span style={actionNameStyle}>{action.data.name}</span>
```
Comment (arunoda) [inline src/modules/ui/components/action_logger/index.js]: @ritz078 When we update the new inspector. We may need to remove this right?
Could you work on that?

Comment (ritz078) [inline src/modules/ui/components/action_logger/index.js]: once the new one is released, i will remove it.

**Pull request merged**